gef-ui/server/controllers/activities-controller/index.test.js

```diff
@@ -1,70 +1,62 @@
 import { fromUnixTime } from 'date-fns';
-import { mapPortalActivities, getPortalActivities } from './index';
-
+import { mapPortalActivities, getPortalActivities } from '.';
 import api from '../../services/api';
-
 import mocks from '../mocks';
 
 jest.mock('../../services/api');
 
-/*
-   ensures that the mapPortalActivities returns an array
-   in the correct format for mojTimeline
-*/
-
-describe('mapPortalActivities', () => {
-  const gefActivity = [
-    {
-      type: 'NOTICE',
-      timestamp: 1638458265,
-      author: {
-        firstName: 'Bob',
-        lastName: 'Smith',
-        _id: 12345,
-      },
-      text: '',
-      label: 'Automatic inclusion notice submitted to UKEF',
-      html: '',
-      facilityType: '',
-      ukefFacilityId: '',
-      facilityId: '',
-      maker: '',
-      checker: '',
+const dealSubmissionActivity = [
+  {
+    type: 'NOTICE',
+    timestamp: 1638458265,
+    author: {
+      firstName: 'Bob',
+      lastName: 'Smith',
+      _id: 12345,
     },
-  ];
-
-  const facilityActivity = [
-    {
-      type: 'FACILITY_STAGE',
-      timestamp: 1638458265,
-      author: {
-        firstName: 'Bob',
-        lastName: 'Smith',
-        _id: 12345,
-      },
-      text: '',
-      label: 'Bank facility stage changed',
-      html: 'facility',
-      facilityType: 'Cash facility',
-      ukefFacilityId: '12345',
-      facilityId: '123456',
-      maker: {
-        firstname: 'Joe',
-        surname: 'Bloggs',
-        id: '12345',
-      },
-      checker: {
-        firstname: 'Bob',
-        surname: 'Smith',
-        id: '4567',
-      },
+    text: '',
+    label: 'Automatic inclusion notice submitted to UKEF',
+    html: '',
+    facilityType: '',
+    ukefFacilityId: '',
+    facilityId: '',
+    maker: '',
+    checker: '',
+  },
+];
+
+const facilityActivity = [
+  {
+    type: 'FACILITY_STAGE',
```

Comment:
can any of this come from a test mock?

Reply:
Potentially. There are at least 4x instances of `type` definitions with hard coded uppercase strings.

I would want these to all be aligned and updated in one go. Do not want to do this in this PR, RE scope creep. In this PR, this array has just been moved to the top.